packages/indy-sdk-to-askar-migration/tests/migrate.test.ts

```diff
@@ -70,38 +60,21 @@ describe('Indy SDK To Askar Migration', () => {
    */
   test('indy-sdk sqlite to aries-askar sqlite fails and restores', async () => {
     const indySdkAndAskarConfig: InitConfig = {
-      label: `indy | indy-sdk sqlite to aries-askar sqlite fails and restores | ${utils.uuid()}`,
+      label: `indy | indy-sdk sqlite to aries-askar sqlite successful migration | e9484a22-6f8c-4e35-88c5-83cc1a7f77b4`,
```

Comment:
Efficient!

Reply:
Not sure if the uuids are still required. I think they were added in an attempt to debug an issue where the wallet already existed, but I don'think this is/should be the fix. Can leave for now though, doesn't hurt.